README.md

```diff
@@ -15,6 +15,8 @@
   </a>
 </p>
 
+> <br/>**Volta is unmaintained.** We recommend migrating to [`mise`](https://mise.jdx.dev/). See [issue #2080](https://github.com/volta-cli/volta/issues/2080).<br/><br/>
```

Comment:
We can use an admonition here. I wish “unmaintained” were not too strong, but it’s accurate at this point. 😮‍💨 

```suggestion
> [!IMPORTANT]
> **Volta is unmaintained.** We recommend migrating to [`mise`](https://mise.jdx.dev/). See [issue #2080](https://github.com/volta-cli/volta/issues/2080).

---

```

This will render like this (with the hrule to set it off from the other content):

> [!IMPORTANT]
> **Volta is unmaintained.** We recommend migrating to [`mise`](https://mise.jdx.dev/). See [issue #2080](https://github.com/volta-cli/volta/issues/2080).